### PR TITLE
Global Prelude bump version to 4.1 (#645852)

### PR DESCRIPTION
**Pull request by ToToL**
Packages:
* libprelude
* libpreludedb
* prelude-manager
* prelude-correlator
* prelude-lml
* prelude-lml-rules
* prewikka

### CODE REVIEW

**Comment by ToToL** [comment]
up

File: dev-libs/libprelude/files/libprelude-4.1.0-fix-python-bindings.patch [hunk 1]
```diff
@@ -0,0 +1,38 @@
+--- a/bindings/python/Makefile.am	2017-07-17 12:03:05.653000000 +0200
```
Comment (mgorny) [inline dev-libs/libprelude/files/libprelude-4.1.0-fix-python-bindings.patch]: This looks identical to the old patch. Please don't introduce duplicate files, and just reuse the old patch.

(same for other packages)

File: app-admin/prelude-manager/prelude-manager-4.1.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools systemd
+
+DESCRIPTION="Bus communication for all Prelude modules"
+HOMEPAGE="https://www.prelude-siem.org"
+SRC_URI="https://www.prelude-siem.org/pkg/src/4.1.0/${P}.tar.gz"
```
Comment (mgorny) [inline app-admin/prelude-manager/prelude-manager-4.1.1.ebuild]: Why not `${PV}`?

File: app-admin/prelude-manager/prelude-manager-4.1.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+)
+
+src_prepare() {
+	default
+
+	eautoreconf
+}
+
+src_configure() {
+	econf \
```
Comment (mgorny) [inline app-admin/prelude-manager/prelude-manager-4.1.1.ebuild]: Please convert all ebuilds to use `myconf`.

File: net-analyzer/prelude-correlator/files/prelude-correlator-4.1.1-fix_bruteforce_rule.patch [hunk 4]
```diff
@@ -0,0 +1,11 @@
+--- a/rules/BruteForcePlugin.py
```
Comment (mgorny) [inline net-analyzer/prelude-correlator/files/prelude-correlator-4.1.1-fix_bruteforce_rule.patch]: Are those upstream patches? Please include link to upstream VCS / some indication where they come from.

File: net-analyzer/prelude-lml/prelude-lml-4.1.0.ebuild [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+SRC_URI="https://www.prelude-siem.org/pkg/src/${PV}/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="icu ssl"
+
+RDEPEND="~dev-libs/libprelude-${PV}
+	dev-libs/libpcre
+	icu? ( dev-libs/icu )
```
Comment (mgorny) [inline net-analyzer/prelude-lml/prelude-lml-4.1.0.ebuild]: Why not `:=`? ICU most definitely breaks ABI on upgrade.

File: net-analyzer/prelude-lml/prelude-lml-4.1.0.ebuild [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+)
+
+src_prepare() {
+	default
+
+	eautoreconf
+}
+
+src_configure() {
+	econf \
```
Comment (mgorny) [inline net-analyzer/prelude-lml/prelude-lml-4.1.0.ebuild]: myconf, please.

File: net-analyzer/prelude-lml-rules/prelude-lml-rules-4.1.0.ebuild [hunk 7]
```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2017 Gentoo Foundation
```
Comment (mgorny) [inline net-analyzer/prelude-lml-rules/prelude-lml-rules-4.1.0.ebuild]: Replace your calendar, please. And use `repoman full`.

File: www-apps/prewikka/files/prewikka-4.1.5-fix_rule_error.patch [hunk 8]
```diff
@@ -0,0 +1,29 @@
+--- a/prewikka/view.py
```
Comment (mgorny) [inline www-apps/prewikka/files/prewikka-4.1.5-fix_rule_error.patch]: List source for those patches as well.

**Pull request closed without merging**